physical/raft/fsm.go

```diff
@@ -232,16 +236,16 @@ func (f *FSM) witnessSnapshot(index, term, configurationIndex uint64, configurat
 		}
 	}
 
-	atomic.StoreUint64(f.latestIndex, index)
-	atomic.StoreUint64(f.latestTerm, term)
+	atomic.StoreUint64(f.latestIndex, metadata.Index)
+	atomic.StoreUint64(f.latestTerm, metadata.Term)
 	f.latestConfig.Store(protoConfig)
 
 	return nil
 }
 
 // Delete deletes the given key from the bolt file.
 func (f *FSM) Delete(ctx context.Context, path string) error {
-	defer metrics.MeasureSince([]string{"raft", "delete"}, time.Now())
+	defer metrics.MeasureSince([]string{"raft_storage", "fsm", "delete"}, time.Now())
```

Comment:
these renames seems like a breaking change?

Reply:
Yes you're correct, however the FSM's Delete, DeletePrefix, and Put functions aren't called during normal vault operation. So these shouldn't have been emitted. Get and List will be a change, but there is already another function that emits these same metrics with more accuracy. Really the name here is a bug since it's scoping the metric to an incorrect system.

Reply:
I went ahead and put the metrics for Get and List back so as not to break any existing dashboards. We are not emitting both the old and new names.